packages/config/src/class-config.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * The base Jetpack configuration class file.
+ *
+ * @package automattic/jetpack-config
+ */
+
+namespace Automattic\Jetpack;
+
+use Automattic\Jetpack\Sync\Actions as Sync_Actions;
+use Automattic\Jetpack\Sync\Main as Sync_Main;
+use Automattic\Jetpack\Plugin\Tracking as Plugin_Tracking;
+use Automattic\Jetpack\Terms_Of_Service;
+
+/**
+ * The configuration class.
+ */
+class Config {
+
+	/**
+	 * The Jetpack class instance.
+	 *
+	 * @var Jetpack
+	 */
+	protected $jetpack;
+
+	/**
+	 * The initial setting values.
+	 *
+	 * @var Array
+	 */
+	protected $config = array(
+		'sync'                => false,
+		'sync_woocommerce'    => false,
+		'sync_wp_super_cache' => false,
+		'tracking'            => false,
+		'tos'                 => false,
+	);
+
+	/**
+	 * Creates the configuration class instance, initalized with the Jetpack object.
+	 *
+	 * @param \Jetpack $jetpack the main object to initalize everything with.
+	 */
+	public function __construct( \Jetpack $jetpack ) {
+		$this->jetpack = $jetpack;
```

Comment:
What do you think of removing the jetpack object from this class? It doesn't appear to be used right now.

Reply:
It looks like it is being used on L114, but my concern is trying to grok how this package is being used.

What other consumer would have a Jetpack object to pass through?